owncloud/owncloud.py

```diff
@@ -878,6 +919,69 @@ def remove_user_from_group(self, user_name, group_name):
 
         raise ResponseError(res)
 
+    def add_user_to_subadminGroup(self, user_name, group_name):
+        """Adds a user to a subadmin group.
+
+        :param user_name:  name of user to be added to subadmin group
+        :param group_name:  name of subadmin group
+        :returns: True if user added
+        :raises: ResponseError in case an HTTP error status was returned
+
+        """
+
+        res = self.__make_ocs_request(
+            'POST',
+            self.OCS_SERVICE_CLOUD,
+            'users/' + user_name + '/subadmins',
```

Comment:
@tomneedham thanks for the info, so we don't need to worry about special characters.

@svigneux I think in general it is still best practice to properly URL-encode/escape parameters set into the URL or the post parameters (in the case of POST the requests library does it automatically).
Do you mind adding code to escape/encode the group name still ?